### PR TITLE
Normalized shortcut formatting for quote expansion control

### PR DESCRIPTION
**Pull request by Sinharitik589**
Fixes vector-im/element-web#19685

Signed-off-by: Ritik Kumar [sinharitik18112835@gmail.com](mailto:sinharitik18112835@gmail.com)

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Normalized shortcut formatting for quote expansion control ([\#7995](https://github.com/matrix-org/matrix-react-sdk/pull/7995)). Fixes vector-im/element-web#19685. Contributed by @Sinharitik589.
 * 
bf_af
Fixes https://github.com/vector-im/element-web/issues/19685

Preview: https://pr7995--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by kerryarchibald** [comment]
Hi @Sinharitik589, please fill out the pull request template with an entry for the changelog and some before and after screenshots so we can easily see the changes. You can check our [contributing guide](https://github.com/matrix-org/matrix-js-sdk/blob/master/CONTRIBUTING.md#how-to-contribute) for more details.

File: src/i18n/strings/fi.json [hunk 1]
```diff
@@ -2656,6 +2656,9 @@
     "Image": "Kuva",
     "Sticker": "Tarra",
     "Expand quotes │ ⇧+click": "Laajenna lainaukset │ ⇧+napsauta",
+    "Expand quotes":"Laajenna lainaukset",
+    "Collapse quotes":"Supista lainaukset",
+    "click":"⇧+napsauta",
```
Comment (t3chguy) [inline src/i18n/strings/fi.json:2661]: Same as above

File: src/i18n/strings/hu.json [hunk 2]
```diff
@@ -3166,6 +3166,9 @@
     "Leave all rooms": "Kilépés minden szobából",
     "Don't leave any rooms": "Ne lépjen ki egy szobából sem",
     "Expand quotes │ ⇧+click": "Idézetek kinyitása │ ⇧+kattintás",
+    "Expand quotes":"Idézetek kinyitása",
+    "Collapse quotes":"Idézetek bezárása",
+    "click":"⇧+kattintás",
```
Comment (t3chguy) [inline src/i18n/strings/hu.json:3171]: Same as above

File: src/components/views/messages/MessageActionBar.tsx [hunk 3]
```diff
@@ -382,9 +382,18 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
                     'mx_MessageActionBar_expandMessageButton': !this.props.isQuoteExpanded,
                     'mx_MessageActionBar_collapseMessageButton': this.props.isQuoteExpanded,
                 });
+                const tooltip = <div>
+                    <div className="mx_Tooltip_title">
+                        { this.props.isQuoteExpanded ? _t("Collapse quotes") : _t("Expand quotes") }
+                    </div>
+                    <div className="mx_Tooltip_sub">
+                        { _t("click") }
```
Comment (t3chguy) [inline src/components/views/messages/MessageActionBar.tsx:390]: The shift part should be in this string, given the string should be an exact english translation (where pluralisation is not involved) , see https://github.com/vector-im/element-web/blob/develop/docs/translating-dev.md

File: src/i18n/strings/ja.json [hunk 4]
```diff
@@ -2441,6 +2441,8 @@
     "Clear room list filter field": "ルーム一覧のフィルターの領域を消去",
     "Expand map": "地図を展開",
     "Expand quotes │ ⇧+click": "引用を展開｜⇧＋クリック",
+    "Expand quotes":"引用を展開",
+    "click":"⇧＋クリック",
```
Comment (t3chguy) [inline src/i18n/strings/ja.json:2445]: Same as above

File: src/i18n/strings/nl.json [hunk 5]
```diff
@@ -3166,6 +3166,9 @@
     "Leave all rooms": "Alle kamers verlaten",
     "Don't leave any rooms": "Geen kamers verlaten",
     "Expand quotes │ ⇧+click": "Quotes uitvouwen │ ⇧+click",
+    "Expand quotes":"Quotes uitvouwen",
+    "Collapse quotes":"Quotes invouwen",
+    "click":"⇧+click",
```
Comment (t3chguy) [inline src/i18n/strings/nl.json:3171]: Same as above

File: src/i18n/strings/pt_BR.json [hunk 6]
```diff
@@ -2973,6 +2973,9 @@
     "Zoom in": "Aumentar zoom",
     "Zoom out": "Diminuir zoom",
     "Expand quotes │ ⇧+click": "Expandir citações | ⇧+click",
+    "Expand quotes":"Expandir citações",
+    "Collapse quotes":"Recolher citações",
+    "click":"⇧+click",
```
Comment (t3chguy) [inline src/i18n/strings/pt_BR.json:2978]: Same as above

File: src/i18n/strings/nl.json (hunk 5 above)
Comment (t3chguy) [inline src/i18n/strings/nl.json:3171]: Same as above

File: src/i18n/strings/ru.json [hunk 7]
```diff
@@ -3152,6 +3152,9 @@
     "You are about to leave <spaceName/>.": "Вы собираетесь покинуть <spaceName/>.",
     "%(reactors)s reacted with %(content)s": "%(reactors)s отреагировали %(content)s",
     "Expand quotes │ ⇧+click": "Развернуть цитаты │ ⇧+нажатие",
+    "Expand quotes":"Развернуть цитаты",
+    "Collapse quotes":"Свернуть цитаты",
+    "click":"⇧+нажатие",
```
Comment (t3chguy) [inline src/i18n/strings/ru.json:3157]: Same as above

File: src/i18n/strings/sq.json [hunk 8]
```diff
@@ -3163,7 +3163,10 @@
     "Leave all rooms": "Braktisi krejt dhomat",
     "Don't leave any rooms": "Mos braktis ndonjë dhomë",
     "Expand quotes │ ⇧+click": "Hapi citimet │ ⇧+klikim",
-    "Collapse quotes │ ⇧+click": "Tkurri citimet │ ⇧+klikim",
+    "Expand quotes":"Hapi citimet",
+    "Collapse quotes":"Tkurri citimet",
+    "click":"⇧+klikim",
+    "Collapse quotes │ ⇧+click": "Tkurri citimet │⇧+klikim",
```
Comment (t3chguy) [inline src/i18n/strings/sq.json:3169]: Same as above

File: src/i18n/strings/sv.json [hunk 9]
```diff
@@ -3185,6 +3185,9 @@
     "Fetching data...": "Hämtar data…",
     "In reply to <a>this message</a>": "Som svar på <a>detta meddelande</a>",
     "Expand quotes │ ⇧+click": "Expandera citat | ⇧+klick",
+    "Expand quotes":"Expandera citat",
+    "Collapse quotes":"Kollapsa citat",
+    "click":"⇧ + klick",
```
Comment (t3chguy) [inline src/i18n/strings/sv.json:3190]: Same as above

File: src/i18n/strings/uk.json [hunk 10]
```diff
@@ -2888,6 +2888,9 @@
     "Invalid file%(extra)s": "Пошкоджений файл%(extra)s",
     "Error decrypting attachment": "Помилка розшифрування вкладення",
     "Expand quotes │ ⇧+click": "Розгорнути цитати │ ⇧+click",
+    "Expand quotes":"Розгорнути цитати",
+    "Collapse quotes":"Згорнути цитати",
+    "click":"⇧ + klik",
```
Comment (t3chguy) [inline src/i18n/strings/uk.json:2893]: Same as above

File: src/i18n/strings/vi.json [hunk 11]
```diff
@@ -1458,6 +1458,9 @@
     "Download %(text)s": "Tải xuống %(text)s",
     "Message Actions": "Hành động tin nhắn",
     "Expand quotes │ ⇧+click": "Mở rộng trích dẫn │ ⇧+click",
+    "Expand quotes":"Mở rộng trích dẫn",
+    "Collapse quotes":"Thu gọn trích dẫn",
+    "click":"⇧ + click",
```
Comment (t3chguy) [inline src/i18n/strings/vi.json:1463]: Same as above

File: src/i18n/strings/zh_Hant.json [hunk 12]
```diff
@@ -3169,6 +3169,9 @@
     "%(reactors)s reacted with %(content)s": "%(reactors)s 使用了 %(content)s 反應",
     "Joining space …": "正在加入空間……",
     "Expand quotes │ ⇧+click": "展開引用 │ ⇧+點擊",
+    "Expand quotes":"展開引用",
+    "Collapse quotes":"折疊引用",
+    "click":"⇧+點擊",
```
Comment (t3chguy) [inline src/i18n/strings/zh_Hant.json:3174]: Same as above

File: src/components/views/messages/MessageActionBar.tsx (hunk 3 above)
Comment (t3chguy) [reply src/components/views/messages/MessageActionBar.tsx:390]: Should probably also have spaces around the `+`

image

**Review by t3chguy — CHANGES_REQUESTED** [review]

**Comment by Sinharitik589** [comment]
image
changed @t3chguy

**Comment by t3chguy** [comment]
You appear to have included a lot of unrelated i18n related changes.

**Comment by Sinharitik589** [comment]
> You appear to have included a lot of unrelated i18n related changes.

My bad :(  ,fixed it now ..

File: src/components/views/messages/MessageActionBar.tsx [hunk 13]
```diff
@@ -382,9 +382,18 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
                     'mx_MessageActionBar_expandMessageButton': !this.props.isQuoteExpanded,
                     'mx_MessageActionBar_collapseMessageButton': this.props.isQuoteExpanded,
                 });
+                const tooltip = <div>
+                    <div className="mx_Tooltip_title">
+                        { this.props.isQuoteExpanded ? _t("Collapse quotes") : _t("Expand quotes") }
+                    </div>
+                    <div className="mx_Tooltip_sub">
+                        { `${_t("Click")} + ⇧` }
```
[the remark this answers is not in the document]
Comment (Sinharitik589) [reply src/components/views/messages/MessageActionBar.tsx]: image

**Review by t3chguy — APPROVED** [review]
This is great, thank you!

**Pull request merged**